data/sql/updates/pending_db_world/rev_1696847020812342700.sql

```diff
@@ -0,0 +1,5 @@
+-- Quest: Grim Message
+DELETE FROM `gameobject` WHERE `id` = 142698;
+INSERT INTO `gameobject` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `position_x`, `position_y`, `position_z`, `orientation`, `rotation0`, `rotation1`, `rotation2`, `rotation3`, `spawntimesecs`, `animprogress`, `state`, `ScriptName`, `VerifiedBuild`) VALUES
+(99872, 142698, 0, 0, 0, 1, 1, -21.77, -2802.7, 121.85, 0, 0, 0, 0.588741, 0.808322, 180, 0, 1, '', 0),
+(99873, 142698, 0, 0, 0, 1, 1, -24.2883, -2468.25, 122.442, 0, 0, 0, 0.588741, 0.808322, 180, 0, 1, '', 0);
```

Comment:
Currently there is another entry for this particular gob, what happened with that one? Also, from where did you get the rotation2 and 3?

Reply:
3rd spawn was incorrect, rot is from trinity, but now that I think of it, it should be 0 (mangos has it at 0) since it's a spellfocus rotation is irrelevant.